faskinner.lic

```diff
@@ -45,13 +45,16 @@ class FaSkinner
   end
 
   def train_skills
-    case bput("skin my #{@trainer}",'You skillfully','A small blue-belly crocodile with prominently','You must be holding','The leather looks frayed','A small fuzzy caracal with tufted ears')
+    case bput("skin my #{@trainer}",'You skillfully','A small blue-belly crocodile with prominently','You must be holding','The leather looks frayed','A small fuzzy caracal with tufted ears','need to have a bladed instrument')
     when /You must be holding/
       get_fred
     when /The leather looks frayed/
       echo ("No Charges!  Try again later.")
       exit
-    end 
+    when /need to have a bladed instrument/
+      echo ("You need to either wear a skinning knife or hold a bladed weapon")
```

Comment:
Instead of echo, use message.

`DRC.message("You need to either wear a skinning knife or hold a bladed weapon")`

This will make the text red on the screen.

Reply:
Yeah, I thought about that too. The existing code uses echo throughout but probably worth updating those to DRC.message too. Thanks